src/algebra/module/equiv.lean

```diff
@@ -652,3 +652,33 @@ rfl
 end add_comm_group
 
 end add_equiv
+
+section linear_map
+
+variables [ring R] [add_comm_group M] [module R M] (T : M →ₗ[R] M)
+
+lemma linear_map.exists_linear_equiv_of_invertible [invertible T] :
+  ∃ S : M ≃ₗ[R] M, T = S :=
+begin
```

Comment:
`invertible` carries data; so I think a better spelling would be a definition and a lemma,

* `linear_equiv.of_invertible : M ≃ₗ[R] M`
* `linear_equiv.coe_of_invertible : of_invertible T = T`